### PR TITLE
add layer for people

### PR DESCRIPTION
**Pull request by SimonLab**
see #569 
Add google data layer.

### CODE REVIEW

File: lib/transforms/search-results-to-template-data.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ module.exports = (queryParams, results, config) => {
   const clearCategoryLink = createClearStickyFilterLink(queryParams, rootUrl, /categories/);
   const clearMuseumLink = createClearStickyFilterLink(queryParams, rootUrl, /museum|gallery/);
   const clearArchiveLink = createClearStickyFilterLink(queryParams, rootUrl, /archive/);
+  const inProduction = config.NODE_ENV === 'production';
```
Comment (SimonLab) [inline lib/transforms/search-results-to-template-data.js]: yes right, thanks for spotting that

**Comment by SimonLab** [comment]
@Danwhy I'm now checking the status of the application "production" or "test" on the json functions step instead of the html step, this means that the config will be always define there, do you mind to have another look at the code, thanks.

**Comment by Danwhy** [comment]
LGTM

File: client/routes/document.js [hunk 2]
```diff
@@ -28,6 +28,10 @@ function load (ctx, next) {
       }
       var data = JSONToHTML(json);
       ctx.state.data = data;
+      if (data.inProduction) {
```
Comment (nelsonic) [inline client/routes/document.js]: is `data.inProduction` based on `NODE_ENV` (_environment variable_) ?

**Comment by nelsonic** [comment]
@SimonLab looks good. 😍 
@Danwhy thanks for init check. 👍

File: client/routes/document.js (hunk 2 above)
Comment (SimonLab) [inline client/routes/document.js]: yes, if NODE_ENV is different to "production" inProduction will be false

**Pull request merged**